api/db/mongoose.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const mongo_uri = 'mongodb://localhost/hyf';
+mongoose.connect(mongo_uri, { useNewUrlParser: true }, function(err) {
+  if (err) {
+    throw err;
+  } else {
+    console.log(`Successfully connected to ${mongo_uri}`);
+  }
+});
+
+module.export = mongoose;
```

Comment:
Add newline, reason https://unix.stackexchange.com/a/18789

```suggestion
module.export = mongoose;

```